src/foundry/client/pixi/groups/effects.d.ts

```diff
@@ -1,29 +1,166 @@
-/**
- * A container group which contains visual effects rendered above the primary group.
- */
-declare class EffectsCanvasGroup extends PIXI.Container {
-  constructor();
+export {};
 
-  walls: WallsLayer;
+type CanvasBackgroundAlterationEffects = unknown;
```

Comment:
I'd say we should put an explicit TODO to reduce the chances of this being lost.

Reply:
It's fixed in the PIXI-Layers branch.